versioned_docs/version-1.2/commands/oras_cp.mdx

```diff
@@ -46,8 +47,7 @@ oras cp -r localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
 Copy an artifact and referrers using specific methods for the Referrers API:
 
 ```bash
-oras cp -r --from-distribution-spec v1.1-referrers-api --to-distribution-spec v1.1-referrers-tag \
-    localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
+oras cp -r --from-distribution-spec v1.1-referrers-api --to-distribution-spec v1.1-referrers-tag     localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
```

Comment:
Looks like you want to rerun this after #352

Reply:
Yeh or if this merges, I'll include the change in #352